source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5618,6 +5664,31 @@ export class ColumnSet extends StylableCardElementContainer {
     private _columns: Array<Column> = [];
     private _renderedColumns: Array<Column>;
 
+    private createColumnInstance(json: any, errors: Array<HostConfig.IValidationError>): Column {
+        return createCardObjectInstance<Column>(
+            this,
+            json,
+            [], // Forbidden types not supported for elements for now
+            (typeName: string) => {
+                return !typeName || typeName === "Column" ? new Column() : null;
+            },
+            (typeName: string, errorType: InstanceCreationErrorType) => {
+                if (errorType == InstanceCreationErrorType.UnknownType) {
+                    return {
+                        error: Enums.ValidationError.UnknownElementType,
+                        message: "Invalid element type " + typeName + " in a ColumnSet. Attempting to fall back."
```

Comment:
> fall back [](start = 101, length = 10)

Nit: "fallback" (one word) to be consistent with the usage of the term elsewhere in this same file.

Reply:
That wouldn't be grammatically correct though. "To fallback" isn't a verb as far as I know.

Reply:
Sure but looks like we have been using "fallback" as a verb - in our docs, in this code file even . My comment is merely to make usage consistent.

---
In reply to: [279438647](https://github.com/Microsoft/AdaptiveCards/pull/2771#discussion_r279438647) [](ancestors = 279438647)